project/src/main/data/player-save-upgrader.gd

```diff
@@ -290,7 +321,7 @@ func _upgrade_38c3(_old_save_items: Array, save_item: SaveItem) -> SaveItem:
 	return save_item
 
 
-func _upgrade_37b3(_old_save_items: Array, save_item: SaveItem) -> SaveItem:
+func _upgrade_3776(_old_save_items: Array, save_item: SaveItem) -> SaveItem:
```

Comment:
Document this in the commit message.

Reply:
Added commit message, `Fixed inconsistent naming of '_upgrade_37b3()' methods ...`